plugin/cache/handler.go

```diff
@@ -83,6 +99,18 @@ func (c *Cache) get(now time.Time, state request.Request, server string) (*item,
 	return nil, false
 }
 
+func (c *Cache) getItem(state request.Request, server string) *item {
```

Comment:
getIgnoreTTL or something? getItem is very simular to get. Might also benefit from some documentation :)

Reply:
I noticed the 2 metrics in the old get method would now never be incremented. Only getIgnoredTTL knows where the item came from (Denial, Success). What would be the right thing to do here?